lib/formatters.js

```diff
@@ -134,14 +134,32 @@ function colorize(color, str) {
  * @deprecated
  */
 function formatTrace(level, context, message, args, err) {
-  args.unshift(
-      templateTrace + message,
-      (new Date()).toISOString(),
-      level,
-      context.corr || notAvailable,
-      context.trans || notAvailable,
-      context.op || notAvailable
-  );
+
+  var recontext = {
+    time: (new Date()).toISOString(),
+    lvl: level,
+    corr: context.corr || notAvailable,
+    trans: context.trans || notAvailable,
+    op: context.op || notAvailable
+  };
+
+  Object.keys(context)
+      .filter(function(key) {
+        return !(context[key] && Object.prototype.toString.call(context[key]) === '[object Function]');
```

Comment:
Do you mean protecting the case where someone passes a `NodeLIst` as the value of a context property (which would be printed calling its `toString()` function) and that same someone is using PhantonJS 1.9 (current version is 2.1)??? :p Uhmmm, backwards compatibility is always tough and a limit has to be imposed somewhere... :p But, OK, I buy it... :)
Anyhow, since the PR is landed, nothing more to say... :)